rq/cli/helpers.py

```diff
@@ -219,11 +220,14 @@ class CliConfig(object):
     """A helper class to be used with click commands, to handle shared options"""
     def __init__(self, url=None, config=None, worker_class=DEFAULT_WORKER_CLASS,
                  job_class=DEFAULT_JOB_CLASS, queue_class=DEFAULT_QUEUE_CLASS,
-                 connection_class=DEFAULT_CONNECTION_CLASS, *args, **kwargs):
+                 connection_class=DEFAULT_CONNECTION_CLASS, path=None, *args, **kwargs):
```

Comment:
Instead of splitting the `path` string by `:`, I think it would be better if we accept multiple `path` arguments like http://click.pocoo.org/5/options/#multiple-values-from-environment-values

So we can invoke RQ like this: `rqworker --path 'foo' --path 'bar'`